From 441e8f2eef71190f49c0e4ce184288699c5ebdc3 Mon Sep 17 00:00:00 2001
From: Ricky CX Wu <ricky.cx.wu.wiwynn@gmail.com>
Date: Fri, 4 Oct 2024 13:59:39 +0800
Subject: [PATCH 2/2] configurations: yosemite4: update threshold for fan speed

Gerrit: 74785

Based on DVT test result from thermal team, the threshold of
temperature sensors on spider board and fan sensors need to be
updated due to low margin.

Change-Id: I4fa2cd66f1da1ee507f2951eea45dbfcf2bf062d
Signed-off-by: Ricky CX Wu <ricky.cx.wu.wiwynn@gmail.com>
---
 ...oard_fsc_max_adc_ti_led_nxp_efuse_max.json | 36 +++++++++----------
 ...oard_fsc_max_adc_ti_led_nxp_efuse_mps.json | 26 +++++++-------
 ...ard_fsc_nct_adc_max_led_ons_efuse_max.json | 36 +++++++++----------
 ...ard_fsc_nct_adc_max_led_ons_efuse_mps.json | 36 +++++++++----------
 .../yosemite4_spiderboard_adc_max_pwr_ti.json | 16 +++++++--
 .../yosemite4_spiderboard_adc_ti_pwr_ti.json  | 16 +++++++--
 6 files changed, 95 insertions(+), 71 deletions(-)

diff --git a/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_max.json b/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_max.json
index 2c2e421..682d7e9 100644
--- a/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_max.json
+++ b/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_max.json
@@ -281,7 +281,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -311,13 +311,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -353,7 +353,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -383,13 +383,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -425,7 +425,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -455,13 +455,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -497,7 +497,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -527,13 +527,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -569,7 +569,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -599,13 +599,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -641,7 +641,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -671,13 +671,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
diff --git a/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json b/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json
index 0d0654f..d46ff99 100644
--- a/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json
+++ b/configurations/yosemite4_fanboard_fsc_max_adc_ti_led_nxp_efuse_mps.json
@@ -281,7 +281,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -311,13 +311,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -383,13 +383,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -455,13 +455,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -527,13 +527,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -599,13 +599,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -671,13 +671,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
diff --git a/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_max.json b/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_max.json
index 9b3a8cb..53b8173 100644
--- a/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_max.json
+++ b/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_max.json
@@ -31,7 +31,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -61,13 +61,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -103,7 +103,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -133,13 +133,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -175,7 +175,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -205,13 +205,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -247,7 +247,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -277,13 +277,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -319,7 +319,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -349,13 +349,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -391,7 +391,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -421,13 +421,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
diff --git a/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json b/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json
index 4f85163..7554571 100644
--- a/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json
+++ b/configurations/yosemite4_fanboard_fsc_nct_adc_max_led_ons_efuse_mps.json
@@ -31,7 +31,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -61,13 +61,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -103,7 +103,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -133,13 +133,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -175,7 +175,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -205,13 +205,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -247,7 +247,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -277,13 +277,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -319,7 +319,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -349,13 +349,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
@@ -391,7 +391,7 @@
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 13140
+                    "Value": 13970
                 },
                 {
                     "Direction": "less than",
@@ -421,13 +421,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 15950
+                    "Value": 17600
                 },
                 {
                     "Direction": "greater than",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 11970
+                    "Value": 14080
                 },
                 {
                     "Direction": "less than",
diff --git a/configurations/yosemite4_spiderboard_adc_max_pwr_ti.json b/configurations/yosemite4_spiderboard_adc_max_pwr_ti.json
index 2cc93a9..cc3a73b 100644
--- a/configurations/yosemite4_spiderboard_adc_max_pwr_ti.json
+++ b/configurations/yosemite4_spiderboard_adc_max_pwr_ti.json
@@ -995,11 +995,17 @@
             "Bus": 14,
             "Name": "SPIDER_0_TEMP_C",
             "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper non recoverable",
+                    "Severity": 4,
+                    "Value": 150
+                },
                 {
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 50
+                    "Value": 60
                 }
             ],
             "Type": "TMP75"
@@ -1009,11 +1015,17 @@
             "Bus": 14,
             "Name": "SPIDER_1_TEMP_C",
             "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper non recoverable",
+                    "Severity": 4,
+                    "Value": 150
+                },
                 {
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 50
+                    "Value": 60
                 }
             ],
             "Type": "TMP75"
diff --git a/configurations/yosemite4_spiderboard_adc_ti_pwr_ti.json b/configurations/yosemite4_spiderboard_adc_ti_pwr_ti.json
index 4b15436..426baff 100644
--- a/configurations/yosemite4_spiderboard_adc_ti_pwr_ti.json
+++ b/configurations/yosemite4_spiderboard_adc_ti_pwr_ti.json
@@ -1005,11 +1005,17 @@
             "Bus": 14,
             "Name": "SPIDER_0_TEMP_C",
             "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper non recoverable",
+                    "Severity": 4,
+                    "Value": 150
+                },
                 {
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 50
+                    "Value": 60
                 }
             ],
             "Type": "TMP75"
@@ -1019,11 +1025,17 @@
             "Bus": 14,
             "Name": "SPIDER_1_TEMP_C",
             "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper non recoverable",
+                    "Severity": 4,
+                    "Value": 150
+                },
                 {
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 50
+                    "Value": 60
                 }
             ],
             "Type": "TMP75"
-- 
2.25.1

